### ISSUE TITLE
Clicking to home page results in extra feed "flash"

### ISSUE THREAD

**Issue by benhalpern** [forem/forem#9079]
Here's the video:

https://streamable.com/m0nfch

**To reproduce:**

- Go to home page
- Click into post
- Click logo to go back to home page

It seems like there is an extra funky render happening most of the time when I do this. Occasionally not.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by nickytonline** [comment]
I’ll take a look into this.

**Comment by benhalpern** [comment]
@nickytonline any learnings on this?

**Comment by nickytonline** [comment]
I wasn’t able to reproduce this behaviour. Even showing my connection down with the dev tools, I was still unable to reproduce this, even in Firefox. I  tried about a dozen times and no issue, not even once. I’m guessing you’re still experiencing the issue?

**Comment by nickytonline** [comment]
There is another issue talking about loading states for the feed not showing up sometimes and I wonder if it’s related. I’ll dig into this tomorrow.

**Comment by nickytonline** [comment]
Found the loading state issue, #9111. I have a hunch it's related.
